WalletWasabi.Gui/Controls/WalletExplorer/SendControlViewModel.cs

```diff
@@ -285,21 +285,21 @@ protected SendControlViewModel(Wallet wallet, string title)
 					{
 						if (!Money.TryParse(AmountText, out Money amount) || amount == Money.Zero)
 						{
-							NotificationHelpers.Warning("Invalid amount.");
+							NotificationHelpers.Warning("Invalid amount.", sender: Wallet);
```

Comment:
All other `Warning` notifications have no title except this one. I think we can take the opportunity and add the `""` for title here too.

Reply:
Do we need these kinds of titles anyway? Warning! Info! We can open a discussion about this in a separate PR.